src/effects/hearts/index.ts

```diff
@@ -0,0 +1,190 @@
+/*
+ Copyright 2021 The Matrix.org Foundation C.I.C.
```

Comment:
Copyright should be yourself (plus anywhere you've taken code from)

Reply:
done!